cypress/support/typeahead.po.ts

```diff
@@ -5,10 +5,91 @@ export class TypeaheadPo extends BaseComponent {
   pageTitle = 'Typeahead';
   ghLinkToComponent = 'https://github.com/valor-software/ngx-bootstrap/tree/development/src/typeahead';
 
-  tagInput = 'input';
-  containerTypeahead = 'typeahead-container';
+  inputSelector = 'input';
+  cardHeader = '.card-header';
+  activeDropdown = '.dropdown';
+  dropdownBtn = '.dropdown button';
+  dropdownItem = '.dropdown-item';
+  formGroup = 'label';
+  dropdownHeader = '.dropdown-header';
+  btnSelector = '.btn';
+  noResultAlert = '.alert-danger';
 
   exampleDemosArr = {
-    reactiveForms: 'demo-typeahead-reactive-form'
+    basic: 'demo-typeahead-basic',
+    itemTemplate: 'demo-typeahead-item-template',
+    optionField: 'demo-typeahead-field',
+    asyncData: 'demo-typeahead-async',
+    withDelay: 'demo-typeahead-delay',
+    templateDriven: 'demo-typeahead-form',
+    reactiveForms: 'demo-typeahead-reactive-form',
+    groupingResults: 'demo-typeahead-grouping',
+    ignoreSpaceAndOrder: 'demo-typeahead-single-world',
+    delimiters: 'demo-typeahead-phrase-delimiters',
+    dropUp: 'demo-typeahead-dropup',
+    onBlur: 'demo-typeahead-on-blur',
+    appendToBody: 'demo-typeahead-container',
+    noResult: 'demo-typeahead-no-result',
+    scrollable: 'demo-typeahead-scrollable',
+    latinize: 'demo-typeahead-latinize',
+    onSelect: 'demo-typeahead-on-select',
+    resultOnBlur: 'demo-typeahead-show-on-blur',
+    configDefaults: 'demo-typeahead-config'
   };
+
+  isSelectedItemDisplayed(baseSelector: string, inputtedText: string, resultToCheck?: string) {
+    if (!resultToCheck) {
+      this.clickOnDropdownItem(this.activeDropdown, inputtedText);
+      this.isDemoContainsTxt(baseSelector, inputtedText);
+    } else {
+      this.clickOnDropdownItem(this.activeDropdown, inputtedText);
+      this.isDemoContainsTxt(baseSelector, resultToCheck);
+    }
+  }
+
+  isElementVisible(baseSelector: string, elementToFind: string, rowNum?: number) {
+    if (!rowNum) {
+      cy.get(`${ baseSelector } ${elementToFind}`).should('be.visible');
```

Comment:
remove if.else, save just       cy.get(`${ baseSelector } ${elementToFind}`).eq(rowNum).should('be.visible');